packages/web-components/src/components/va-date/va-date.scss

```diff
@@ -2,77 +2,70 @@
 @import '../../mixins/uswds-error-border.scss';
 @import '../../mixins/form-field-error.css';
 @import '../../mixins/hint-text.css';
-
-.usa-label--required {
-  color: var(--vads-color-secondary-dark);
-}
-
-.usa-error-message {
-  color: var(--vads-color-secondary-dark);
-}
-
-
-/** Original Component Style **/
+@import 'uswds-core/src/styles/mixins/helpers/add-label-styles';
 
 :host {
   display: block;
   font-family: var(--font-source-sans);
   margin-top: 1.875rem;
 }
 
+#error-message {
+  margin-bottom: 0;
+}
+
+// TODO: Update markup so that can import and use usa-fieldset styles
 fieldset {
   border: none;
   margin: 0;
   padding: 0;
 }
 
+// TODO: Update markup so that can import and use usa-legend styles more directly
 legend {
-  border: 0;
-  font-size: 1.06rem;
-  padding: 0;
+  @include add-label-styles;
 }
 
-span.required {
-  color: var(--vads-color-secondary-dark);
+legend,
+va-select,
+va-text-input {
+  font-size: var(--vads-font-size-source-sans-normalized);
 }
 
-#error-message {
+span.required {
   color: var(--vads-color-secondary-dark);
-  display: block;
-  margin-bottom: 0;
 }
 
 .date-container {
+  column-gap: var(--units-2);
   display: flex;
-  margin-top: 12px;
+  margin-top: var(--units-1p5);
   flex-wrap: wrap;
-  row-gap: 12px;
+  row-gap: var(--units-1p5);
 }
 
 :host([error]:not([error=''])) .date-container {
   padding-top: 0;
 }
 
 .select-month {
-  width: 8.125rem;
-  margin-right: 0.9375rem;
+  width: 16ch;
```

Comment:
> [!note]
> Replaced `rem` with `ch` here because the width is determined primarily by the number of characters expected. I did not use the current input width tokens because they rely on `ex` units, which is intended primarily for vertical height of the letters, rather than width.